target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -22,6 +22,11 @@ ath79_setup_interfaces()
 	mikrotik,routerboard-wapr-2nd)
 		ucidef_set_interface_lan "eth0"
 		;;
+	mikrotik,routerboard-952ui-5ac2nd)
+		ucidef_set_interface_wan "eth1"
```

Comment:
Not defining lan (using `ucidef_set_interfaces_lan_wan`) will probably break MAC address assignment, as in https://github.com/openwrt/openwrt/pull/4967#issuecomment-1014381153

Can you check if this is the case?